rpcclient/wallet.go

```diff
@@ -2317,3 +2317,41 @@ func (c *Client) GetInfo() (*btcjson.InfoWalletResult, error) {
 // DUMP
 // importwallet (NYI in btcwallet)
 // dumpwallet (NYI in btcwallet)
+
+// FutureImportMultiResult is a future promise to deliver the result of an
+// ImportMultiAsync RPC invocation (or an applicable error).
+type FutureImportMultiResult chan *response
```

Comment:
This is under the header `Miscellaneous Functions` in `wallet.go` file. I think `importmulti` should fall under the hear `Dump/Import Functions` which is lines 2060-2263.

Within a header each command seems to be in alphabetical order. In this fashion the `importmulti` functions will fall between the `importaddress` and `importprivkey` functions.

Reply:
Thanks. Will keep in mind in future.